File: src/main/java/com/mizule/server/utils/UserResponse.java
package com.mizule.server.utils;

import com.mizule.server.models.Users;
import com.mizule.server.models.Zulespot;

public class UserResponse {
    private Users user;
    private Zulespot zulespot;

    public UserResponse(Users user, Zulespot zulespot) {
        this.user = user;
        this.zulespot = zulespot;
    }

    public Users getUser() {
        return user;
    }

    public void setUser(Users user) {
        this.user = user;
    }

    public Zulespot getZulespot() {
        return zulespot;
    }

    public void setZulespot(Zulespot zulespot) {
        this.zulespot = zulespot;
    }
}

File: app/src/main/java/com/mizule/mizule/utils/UserResponse.kt
package com.mizule.mizule.utils

import com.mizule.mizule.dataClass.zulespotDataClass.Zulespot
import com.mizule.mizule.dataClass.userDataClass.User

data class UserResponse(var user:User,
                   var zulespot:Zulespot) {
}